docs/pipelineruns.md

```diff
@@ -912,6 +912,9 @@ workspaces:
     subPath: my-subdir
 ```
 
+`workspaces[].subPath` can be an absolute value or can reference `pipelineRun` context variables, such as,
+`$(context.pipelineRun.name)` or `$(context.pipelineRun.uid)`.
```

Comment:
since we support 4 replacements:	"$(context.pipelineRun.name)", "$(context.pipeline.name)", "$(context.pipelineRun.namespace)", "$(context.pipelineRun.uid)" 

In this PR's use case, does it only use the 2 mentioned in the doc or all of them are valid use cases?

Reply:
hey @Yongxuanzhang, all of them are supported and valid uses if the pipelineRun author would like to utilize them. But the `name` and `uid` of the `pipelineRun` are two common and meaningful metadata in creating such path for workspaces. There could be others who would like to use name of the pipeline and namespace depending on their use cases. They could be used in conjunction with other context variables for example: `subPath: $(context.pipelineRun.uid)/$(context.pipelineRun.name)`